hackclub.com.yaml

```diff
@@ -763,6 +763,10 @@ cecclphs:
   - ttl: 3600
     type: TXT
     value: google-site-verification=_c3oOWsKBUxlmaanXeqC1tBTLNZGflwsEMQQiuP-EUM
+celestial:
+  - ttl: 600
+    type: CNAME
+    value: https://emma-x1.github.io/Toronto-DoS-Celestial/
```

Comment:
```suggestion
    value: emma-x1.github.io
```

Reply:
I just made the change! I just designed the site myself on my personal github and then Zenab told me to add it through this, if I need to make other changes please let me know!